mesonbuild/backend/ninjabackend.py

```diff
@@ -2303,6 +2311,17 @@ def generate_dynamic_link_rules(self):
 
                 options = self._rsp_options(compiler)
                 self.add_rule(NinjaRule(rule, command, args, description, **options, extra=pool))
+            if mesonlib.is_aix(): 
```

Comment:
Same issue as above

Reply:
This is done. Kindly check the latest commit